pandas/core/generic.py

```diff
@@ -2458,8 +2458,12 @@ def to_msgpack(self, path_or_buf=None, encoding='utf-8', **kwargs):
         return packers.to_msgpack(path_or_buf, self, encoding=encoding,
                                   **kwargs)
 
-    def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
-               index_label=None, chunksize=None, dtype=None, method=None):
+    def to_sql(self, name: str, con,
```

Comment:
I left `con=` as it is. I'm not aware if it's possible to annotate type for external libraries and not create dependencies.